pytorch/training/docker/1.4.0/py3/example/Dockerfile.gpu

```diff
@@ -6,6 +7,6 @@ FROM $BASE_IMAGE
 RUN pip install tensorboardX==1.6.0
 
 # Add any script or repo as required
-RUN cd /var && wget https://raw.githubusercontent.com/kubeflow/pytorch-operator/master/examples/mnist/mnist.py
+COPY mnist.py /var/mnist.py
```

Comment:
Why this change?

Reply:
This is introduced since PT 1.5. PT team change the script and we now maintain it as a separate artifact  https://github.com/aws/deep-learning-containers/blob/master/pytorch/training/docker/build_artifacts/mnist.py

Reply:
Is this necessary now that we are building a PT 1.4 image?

Reply:
Talked with Tushar. We think it should be a general change and should also apply to PT1.4. @TusharKanekiDey